lib/channel.js

```diff
@@ -0,0 +1,66 @@
+var WebSocketLib = require('ws');
+var util         = require('./util');
+
+function createChannel(port) {
```

Comment:
Yeah, I'd prefer if you change it. It's certainly not for performance reasons, I just think that it's a more understandable way to describe what the code is doing.